DATA/SConscript

```diff
@@ -151,33 +307,41 @@ def getVerifyCmd(stage, expList, pipelineFile):
         List of exposure ids to use in generating the calibration.
     pipelineFile : `str`
         Name of pipeline yaml to use.
+    legacyDate: `str`, optional
+        Date string for legacy pipelines.
 
     Returns
     -------
     cmd : `str`
         The constructed command.
     """
-    inputCollections = 'LATISS/raw/all,LATISS/calib,calib/v00'
-    if stage in ('ptc'):
+    inputCollections = "LATISS/raw/all,LATISS/calib,calib/v00"
+    if stage in ("ptc"):
         inputCollections = f"ci_cpp_ptc,{inputCollections}"
 
-    pipelineYaml = os.path.join(PKG_ROOT, "pipelines", pipelineFile)
+    pipelineYaml = os.path.join(PKG_ROOT, "pipelines", f"LATISS_legacy_{legacyDate}", pipelineFile)
```

Comment:
I think this should be `"LATISS", f"legacy_{legacyDate}"` so the `ci_cpp_gen3` override pipelines match the paths of the `cp_pipe` and `cp_verify` legacy pipelines.  This will require `runIsrLSST.yaml` to be moved into a `LATISS` subdirectory, but that will make extensions to different camera datasets easier in the future.